### ISSUE TITLE
Buttons being used as navigation tools

### ISSUE THREAD

**Issue by LocalNewsTV** [bcgov/nr-sustainment-capstone-2024#197]
## Describe the Bug
Buttons are being used to traverse between pages, instead of just being used to perform functions
anchor tags `<a>` should be used to traverse pages, not buttons. It lets screen readers / users know that links are links.

https://theadminbar.com/accessibility-weekly/buttons-vs-links/

## Resolution
Make a second component that for links, you can style it to look like your button, that's 100% ok. 
This also reduces the number of props and overall complexity of your existing button. It has a lot going on!
Alternatively, a React component can have two return statements, you could check if your path value exists, and return a styled `<a>` tag instead. 

## Priority
> I like raising my priorities, before these issues become bigger ones :) 

- [ ] Low
- [ ] Medium
- [x] High
